tests/update/prepare_system_for_update_tests.pm

```diff
@@ -24,6 +24,8 @@ sub run() {
     assert_screen('xterm-started');
     assert_script_sudo "chown $testapi::username /dev/$testapi::serialdev";
     assert_script_sudo "echo \"download.use_deltarpm = false\" >> /etc/zypp/zypp.conf";
+    # Refresh repodata - we use zypper, as pkcon might not be there. But they share metadata anyway
+    assert_script_sudo "zypper refresh";
```

Comment:
hmm.. that will turn out to be fun: we can't use pkcon, as some tests don't have PackageKit installed (minimal tests for example) - we can't use zypper because PK might be blocking it...

So we really finally need to fix PK's locking mechanism aganst zypp (zypp was fixed to actually allow it like 2 weeks ago. Before it was 'load the lib, lock the stack - now is load the lib and lock when you say to lock it with a chance to unlock it'